### PR TITLE
Run integration and build on push to main

### PR DESCRIPTION
**Pull request by brunns**
## Things to check

- [ ] I have added any new ENV vars in all deployed environments
- [ ] I have tested any code added or changed
- [ ] I have run integration tests

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -49,7 +53,10 @@ test-django: stop ## Test django-app
 	docker compose run --no-deps django-app venv/bin/pytest tests/ --ds redbox_app.settings -v --cov=redbox_app.redbox_core --cov-fail-under 80 -o log_cli=true
 
 .PHONY: test-integration
-test-integration: stop run ## Run all integration tests
+test-integration: rebuild run test-integration-without-build ## Run all integration tests
```
Comment (gecBurton) [inline Makefile:56]: :+1:

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -1,9 +1,12 @@
 
 name: build
 on:
+  # On-demand
+  workflow_dispatch:
+  # or on push to main
   push:
     branches:
-      - "**"
+      - 'main'
```
Comment (gecBurton) [inline .github/workflows/build.yml:9]: :+1:

Comment (gecBurton) [inline .github/workflows/build.yml:9]: looks sensible but keen to get @252afh 's views

File: .github/workflows/integration.yml [hunk 3]
```diff
@@ -9,6 +9,10 @@ on:
     - cron: '0 6,12,18 * * 1-5'
   # or on-demand
   workflow_dispatch:
+ # or on push to main
+  push:
+    branches:
+      - 'main'
```
Comment (gecBurton) [inline .github/workflows/integration.yml:15]: :+1:

**Review by 252afh — APPROVED** [review]

**Pull request merged**